packages/react-devtools-extensions/src/main.js

```diff
@@ -210,23 +209,24 @@ function createPanelIfReactLoaded() {
 
         render = (overrideTab = mostRecentOverrideTab) => {
           mostRecentOverrideTab = overrideTab;
-
-          root.render(
-            createElement(DevTools, {
-              bridge,
-              browserTheme: getBrowserTheme(),
-              componentsPortalContainer,
-              enabledInspectedElementContextMenu: true,
-              loadHookNames: parseHookNames,
-              overrideTab,
-              profilerPortalContainer,
-              purgeCachedHookNamesMetadata: purgeCachedMetadata,
-              showTabBar: false,
-              store,
-              warnIfUnsupportedVersionDetected: true,
-              viewAttributeSourceFunction,
-              viewElementSourceFunction,
-            }),
+          import('./parseHookNames').then(
```

Comment:
I'm assuming this async hop is fine here?

Reply:
I think it's OK for now, (waiting on @bvaughn for confirmation). It's a temporary solution to unblock Firefox anyway until we make a suspense cache for modules (which we can use to block the hook names feature for devtools inline).

Reply:
@lunaruan yeah I think the async hop is probably fine? i think the thing that's missing is that before we were calling `root.render(createElement(...))`, but now we're not calling `root.render` at all anymore (unless i'm missing something), so I'm not sure if that will break things?

Reply:
Ah good catch! :D I accidentally deleted that while cleaning up the code before putting up the PR. My bad!

Reply:
I think it's fine, especially since we're planning on changing it